tcmu-runner.service

```diff
@@ -4,5 +4,5 @@ Description=LIO Userspace-passthrough daemon
 [Service]
 Type=dbus
 BusName=org.kernel.TCMUService1
+KillMode=process
```

Comment:
It did work at some point. See Andy's explanation twords the bottom of this email:

https://www.spinics.net/lists/target-devel/msg14773.html

It also might work on your tests now if you do not have a high load and do not fail too many times. If on each run you are not using the entire ring, then when you restart runner, the available ring memory will just be less than the previous run. If you run the tests enough times or were using the entire ring, you will get "tcmu: command timed out" errors and failed commands, because tcmu_queue_cmd_ring is timing out wait for memory to free.